src/sinks/util/rusoto.rs

```diff
@@ -49,6 +54,13 @@ impl AwsCredentialsProvider {
             Ok(Self::Default(creds))
         }
     }
+
+    pub fn new_minimal<A: Into<String>, S: Into<String>>(access_key: A, secret_key: S) -> Self {
```

Comment:
minor nit: I would probably use `impl Trait` here, I find it much nicer to the eyes.